### PR TITLE
Fix broken package

### PR DESCRIPTION
**Pull request by bluurn**
Fix for issue: https://github.com/rgbkrk/atom-achievements/issues/19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
...
...The {Integer} number of points\n  :iconURL     - The {String} URL of an icon to display for the user, which\n                 can be base64 encoded but must have a valid data prefix\n                 such as \"data:image/png;base64,\". Optional, defaults to\n                 spinning octocat.\n```\n\nAchievements are stored under key `{package}:{name}` within an associative array\nof unlocked achievements.\n\n## Configuration\n\nSet how long the achievements banner lasts by changing the notice delay (in milliseconds).\n\n![](https://f.cloud.github.com/assets/836375/2424719/6a9ca422-abab-11e3-925c-3a85f87b3bb1.png)\n\nEmit events from your own packages to grant achievements to your users!\n\n## ROADMAP\n\n* Use the swirling octocat icon :white_check_mark:\n* Populate achievement bar using a message :white_check_mark:\n* Time achievements display out :white_check_mark:\n* Add configuration for timer :white_check_mark:\n* Create an event spec for triggering achievements :white_check_mark:\n* Create some default achievements :white_check_mark:\n* Store achievements so they're only earned once. (per user, per project?) :white_check_mark:\n* Allow for a custom icon :white_check_mark:\n",
```
Comment (rgbkrk) [inline package.json]: Whoa, I never noticed this was here before.

**Comment by rgbkrk** [comment]
OMG thanks for fixing this. Adding you as a maintainer, I'll go ahead and ship a release.

**Comment by bluurn** [comment]
@rgbkrk NP :) just wanted to see the working package

**Comment by rgbkrk** [comment]
```
$ apm publish minor
Preparing and tagging a new version ✓
Pushing v0.9.0 tag ✓
Publishing achievements@v0.9.0 ✓
```

This is awesome. I'm now using achievements again.

**Comment by bluurn** [comment]
:tada:

**Pull request merged**